app/uk/gov/hmrc/cdsreimbursementclaim/models/claim/CompleteClaim.scala

```diff
@@ -45,7 +45,8 @@ final case class CompleteClaim(
   scheduledDocumentAnswer: Option[ScheduledDocumentAnswer],
   reimbursementMethodAnswer: Option[ReimbursementMethodAnswer],
   typeOfClaim: Option[SelectNumberOfClaimsAnswer],
-  associatedMRNsAnswer: Option[List[MRN]]
+  associatedMRNsAnswer: Option[List[MRN]],
+  maybeAssociatedMRNsClaimsAnswer: Option[List[ClaimedReimbursementsAnswer]]
```

Comment:
Maybe it should be Option[NonEmptyList[...]]
because List can be empty anyways?

Reply:
Does not really make sense